opal/mca/patcher/base/patcher_base_patch.c

```diff
@@ -138,10 +142,27 @@ static inline void apply_patch (unsigned char *patch_data, uintptr_t address, si
 {
     ModifyMemoryProtection (address, data_size, PROT_EXEC|PROT_READ|PROT_WRITE);
     memcpy ((void *) address, patch_data, data_size);
-    for (size_t i = 0 ; i < data_size ; i += 16) {
+#if HAVE___CLEAR_CACHE
+    /* do not allow global declaration of compiler intrinsic */
+    void __clear_cache(void* beg, void* end);
+
+    __clear_cache ((void *) address, (void *) (address + data_size));
+#else
+    size_t offset_jump = 16;
+
+#if OPAL_ASSEMBLY_ARCH == OPAL_ARM64
+    offset_jump = 32;
+#endif
+
+    /* align the address */
+    address &= ~(offset_jump - 1);
+
+    for (size_t i = 0 ; i < data_size ; i += ofset_jump) {
```

Comment:
Typo? 
ofset_jump should be offset_jump

Reply:
Yup. Typo. Thanks for catching that.